ggml/src/CMakeLists.txt

```diff
@@ -225,7 +225,7 @@ add_library(ggml
 target_link_libraries(ggml PUBLIC ggml-base)
 
 if (CMAKE_SYSTEM_NAME MATCHES "Linux")
-    target_link_libraries(ggml PRIVATE dl)
+    target_link_libraries(ggml PRIVATE dl stdc++fs)
```

Comment:
We'd need to check for GCC vs Clang or more specifically libstdc++ vs libc++.

Reply:
check CMAKE_CXX_COMPILER_ID is GNU